### PR TITLE
tiny fix

### PR DESCRIPTION
**Pull request by arunahk**
comment result

### CODE REVIEW

**Comment by arunahk** [comment]
I fix failing test and remove a deprecated method in test

File: tests/unit/FunctionInjectorTest.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ public function testTemplate()
     public function testSave()
     {
         $this->funcInjector->save();
-        exec('php -l '.$this->getFileName(), $output, $code);
+        exec('php -l '.__FILE__, $output, $code);
```
Comment (sergeyklay) [inline tests/unit/FunctionInjectorTest.php]: Could you please explain this a bit more?

Comment (arunahk) [inline tests/unit/FunctionInjectorTest.php]: Hi, that was wrong. I fix it below commit. Initially, it should be `$this->funcInjector->getFileName()`

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**